app/code/Magento/Checkout/Test/Mftf/Test/ClearShoppingCartEnableDisableConfigurationTest.xml

```diff
@@ -0,0 +1,91 @@
+        </before>
+        <after>
+            <deleteData createDataKey="createCategory" stepKey="deleteCategory"/>
+            <deleteData createDataKey="createProduct1" stepKey="deleteProduct1"/>
+            <deleteData createDataKey="createProduct2" stepKey="deleteProduct2"/>
+
+            <!-- Disable clear shopping cart -->
+            <magentoCLI command="config:set {{DisableClearShoppingCart.path}} {{DisableClearShoppingCart.value}}" stepKey="disableClearShoppingCart"/>
+            <actionGroup ref="AdminLogoutActionGroup" stepKey="logout"/>
+        </after>
+
+        <!-- Navigate to sales checkout cart configuration -->
+        <actionGroup ref="AdminOpenSalesCheckoutConfigPageActionGroup" stepKey="openSalesCheckoutCartConfig1">
+            <argument name="tabGroupAnchor" value="#checkout_cart-link"/>
+        </actionGroup>
+
+        <!-- Enable clear shopping cart button -->
+        <actionGroup ref="AdminSelectClearShoppingCartConfigurationActionGroup" stepKey="enableClearShoppingCartButton"/>
+        <actionGroup ref="SaveStoreConfigurationActionGroup" stepKey="saveStoreConfiguration1"/>
+        <actionGroup ref="CliCacheCleanActionGroup" stepKey="cliCacheClean1">
+            <argument name="tags" value=""/>
+        </actionGroup>
+
+        <!-- Open product 1 and add to cart -->
+        <actionGroup ref="StorefrontOpenProductEntityPageActionGroup" stepKey="openProduct1Page1">
+            <argument name="product" value="$$createProduct1$$"/>
+        </actionGroup>
+        <actionGroup ref="StorefrontAddToTheCartActionGroup" stepKey="product1AddToCart"/>
+
+        <!-- Open product 2 and add to cart -->
+        <actionGroup ref="StorefrontOpenProductEntityPageActionGroup" stepKey="openProduct2Page">
+            <argument name="product" value="$$createProduct2$$"/>
+        </actionGroup>
+        <actionGroup ref="StorefrontAddToTheCartActionGroup" stepKey="product2AddToCart"/>
+
+        <!-- Go to shopping cart page -->
+        <actionGroup ref="StorefrontCartPageOpenActionGroup" stepKey="openShoppingCartPage1"/>
+
+        <!-- Clear shopping cart -->
+        <actionGroup ref="StorefrontClearShoppingCartActionGroup" stepKey="clearShoppingCart"/>
+        <actionGroup ref="AssertMiniCartEmptyActionGroup" stepKey="assertMiniCartEmpty"/>
+
+        <!-- Return to Admin to disable clear shopping cart -->
+        <actionGroup ref="AdminOpenSalesCheckoutConfigPageActionGroup" stepKey="openSalesCheckoutCartConfig2"/>
+        <actionGroup ref="AdminSelectClearShoppingCartConfigurationActionGroup" stepKey="disableClearShoppingCartButton">
+            <argument name="value" value="{{DisableClearShoppingCart.textValue}}"/>
+        </actionGroup>
+        <actionGroup ref="SaveStoreConfigurationActionGroup" stepKey="saveStoreConfiguration2"/>
+        <actionGroup ref="CliCacheCleanActionGroup" stepKey="cliCacheClean2">
+            <argument name="tags" value=""/>
```

Comment:
edit to only clean the page cache (assuming the test passes w/ only cleaning the page cache)